blogs/cli/index.md

```diff
@@ -0,0 +1,444 @@
+
+For the purepose of this article, the following commands assume you are using `oc`.
+
+### The krew way
+
+```
+Krew itself is a kubectl plugin that is installed and updated via Krew (yes, Krew self-hosts).
+```
+
+Follow the instructions on [https://krew.sigs.k8s.io/docs/user-guide/setup/install/](krew.sigs.k8s.io) official website.
+Once installed, run the following command to install netobserv:
+```
+oc krew install netobserv
+```
+
+### From tar archive
+
+Download the latest version either from [github releases]((https://github.com/netobserv/network-observability-cli/releases)) or [mirror.openshift.com](https://mirror.openshift.com/pub/openshift-v4/amd64/clients/).
+
+Then follow the instructions on [Installing and using CLI plugins](https://docs.openshift.com/container-platform/4.15/cli_reference/openshift_cli/extending-cli-plugins.html#cli-installing-plugins_cli-extend-plugins) from Openshift documentation.
+
+These should resume as following commands using `/usr/local/bin` as PATH folder:
+```
+tar -xvzf ./netobserv-linux-amd64.tar.gz
+chmod +x ./netobserv-linux-amd64/build/oc-netobserv
+sudo mv  ./netobserv-linux-amd64/build/oc-netobserv /usr/local/bin/.
+```
+
+## Verify the installation
+
+To ensure the installation completed correctly, simply run the following command:
+```
+oc netobserv version
+```
+
+It should print `Netobserv CLI version XXX` with the version you picked earlier.
+
+## Usage
+
+Use the following command to display the help:
+```
+oc netobserv help
+```
+
+```
+Netobserv allows you to capture flow and packets from your cluster.
+Find more information at: https://github.com/netobserv/network-observability-cli/
+
+Syntax: netobserv [flows|packets|cleanup] [filters]
+
+options:
+  flows      Capture flows information. Use options below for features / filtering.
+        Options:
+          --interfaces: interfaces to monitor
+          --enable_pktdrop: enable packet drop (default: false)
+          --enable_dns: enable DNS tracking (default: false)
+          --enable_rtt: enable RTT tracking (default: false)
+          --enable_filter: enable flow filter (default: false)
+          --direction: flow filter direction
+          --cidr: flow filter CIDR (default: 0.0.0.0/0)
+          --protocol: flow filter protocol
+          --sport: flow filter source port
+          --dport: flow filter destination port
+          --port: flow filter port
+          --sport_range: flow filter source port range
+          --dport_range: flow filter destination port range
+          --port_range: flow filter port range
+          --icmp_type: ICMP type
+          --icmp_code: ICMP code
+          --peer_ip: peer IP
+          --action: flow filter action (default: Accept)
+  packets    Capture packets from a specific protocol/port pair such as 'netobserv packets tcp,80'.
+  cleanup    Remove netobserv components.
+  version    Print software version.
+  ```
+
+As you can see, the CLI offers two captures options for now:
```

Comment:
```suggestion
As you can see, the CLI offers two capture options for now:
```

Reply:
The two options "flows" and "packets" are presented too far apart (packet being listed much below), and it is mixed up with another bullet list (just after "This will write data into two separate files").

I think it could be rewritten with sub-sections, such as:

```suggestion
As you can see, the CLI offers two capture options for now: `flows` and `packets`.

### oc netobserv flows

[...]

### oc netobserv packets

[...]

```